usaspending_api/awards/models.py

```diff
@@ -191,6 +191,7 @@ class Award(DataSourceTrackedModel):
     create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
     update_date = models.DateTimeField(auto_now=True, null=True)
     latest_submission = models.ForeignKey(SubmissionAttributes, null=True)
+    latest_transaction = models.ForeignKey("awards.Transaction", related_name="latest_transaction", null=True)
```

Comment:
The `related_name` is the fieldname from the `Transaction` side, so it's confusing to have that named `latest_transaction` when it actually points back to this award:

`In [4]: Transaction.objects.first().latest_transaction.first()
Out[4]: <Award: Definitive Contract piid: SBAHQ16C0024 fain: None uri: None>`

Not sure what would be a clearer name - `latest_transaction_of`, `latest_transaction_of_award`, or even `latest_transaction_of_which_award`, maybe?  Alternately, maybe it's not necessary to have a `related_name` here, since  there's already a link from the Transaction up to the parent Award via the Transaction's `.award`:

`
In [5]: Transaction.objects.first().award
Out[5]: <Award: Definitive Contract piid: SBAHQ16C0024 fain: None uri: None>`

... the only utility I can figure is if we want to be able to determine from a Transaction whether it is the last transaction of its parent award.  Do we?